src/model/request/EthMintErc721.ts

```diff
@@ -11,4 +12,6 @@ export class EthMintErc721 extends MintErc721 {
     @Type(() => Fee)
     @ValidateNested()
     public fee?: Fee;
+    public authorAddresses?: string[];
```

Comment:
should be marked as @IsOptional(). Also with conditional validation - if cashbackValues are defined, also authorAddresses should be defined